spec/teaspoon/server_spec.rb

```diff
@@ -76,6 +77,14 @@
       expect(socket).to receive(:close)
       subject.responsive?
     end
+
+    it "checks a port in a given host to see if a server is running" do
+      subject.host = '0.0.0.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.